tests/deb/install.sh

```diff
@@ -19,7 +19,8 @@ prepare_local_apt() {
     if [ -n "${1}" ]; then
         dir="${1}"
     fi
-    if [ -z "$(ls ${dir}/*.deb)" ]; then
+
+    if [ -z "$(ls ${dir}/*/*.deb)" ]; then
```

Comment:
Is this needed after cff69bec7cc63ce16633427c40a779272a0343f1?

Reply:
Good catch, thanks.